perllib/FixMyStreet/Cobrand/Camden.pm

```diff
@@ -92,4 +92,24 @@ sub is_problem_anonymous { 1 }
 
 sub is_comment_anonymous { 1 }
 
+sub social_auth_enabled {
+    my $self = shift;
+
+    return $self->feature('oidc_login') ? 1 : 0;
+}
+
+sub user_from_oidc {
+    my ($self, $payload) = @_;
+
+    # TODO: Remove this once we know which fields the name/email are in.
+    use Data::Dumper;
+    $self->{c}->log->debug("Camden OIDC payload: " . Dumper($payload));
```

Comment:
Are these lines still required?

Reply:
For the time being yes, because we don't yet know which fields the email/name will be in. This logging makes it easier to see what's in the payload, so when the email/name are there we can easily tell what fields they're in.